targets/x86-64

```diff
@@ -1,6 +1,9 @@
 include 'x86.inc'
 
-factory_image('x86-64', 'combined-squashfs', '.img.gz')
-factory_image('x86-64', 'combined-squashfs', '.vdi')
-factory_image('x86-64', 'combined-squashfs', '.vmdk')
-sysupgrade_image('x86-64', 'combined-squashfs', '.img.gz')
+packages {
+        'kmod-gpio-nct5104d',
+        'kmod-leds-gpio',
+        'kmod-pcengines-apuv2',
+}
+
+device('x86-64', 'generic')
```

Comment:
By switching to the proper device handlers, we can also change the
```
try_config('TARGET_MULTI_PROFILE', true)
try_config('TARGET_PER_DEVICE_ROOTFS', true)
```
to `config(...)` in `targets/generic`.